variables.tf

```diff
@@ -41,6 +41,11 @@ variable "root_volume_type" {
   default     = "gp2"
 }
 
+variable "extra_volumes" {
+  description = "Extra volumes for each instance"
```

Comment:
if we specify `ebs_name_format` as variable i would suggest naming this `extra_ebs_volumes`

both new variables need to be specified in the variables sheet

Reply:
I'll use `extra_volumes` for consistency as I saw `root_volume_size` and `root_volume_type` are there. I'll rename `ebs_name_format` to `extra_volume_name_format`